### ISSUE TITLE
`docker compose up` is not enough to remain active, in `examples/full-stack`

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-containers#72]

**Comment by tcompa** [comment]
```
[+] Running 2/0
 ✔ Container fractal-db  Running                                                                                                                                                                                                                          0.0s 
 ✔ Container slurm       Created                                                                                                                                                                                                                          0.0s 
Attaching to fractal-db, fractal-web, server-config, slurm
slurm          | 
slurm          | Starting Slurm services...
slurm          | 
slurm          |  * Starting MUNGE munged
slurm          |    ...done.
slurm          |  * Starting slurm central management daemon slurmctld
slurm          |    ...done.
slurm          |  * Starting OpenBSD Secure Shell server sshd
slurm          |    ...done.
slurm          | 
slurm          | PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
slurm          | main*        up   infinite      1   idle slurmnode1
slurm          | 
slurm          | 
slurm exited with code 0
Gracefully stopping... (press Ctrl+C again to force)
dependency failed to start: container slurm exited (0)
```

**Comment by tcompa** [comment]
cc @apasto and @lorenzocerrone

It turns out that my suggestion to simply run `docker compose up` from `examples/full-stack` was wrong. This will be fixed soon (ref #73).
